crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -119,7 +119,7 @@ function CrowdsourceHinter(runtime, element, data){
      * @param student_answer is the first incorrect answer submitted by the student
      */
     function showStudentHintRatingUX(hint, student_answer){
-        var hintRatingUXTemplate = $(Mustache.render($('#show_hint_rating_ux').html(), {hintText: hint}));
+        var hintRatingUXTemplate = $(Mustache.render($('#show_hint_rating_ux').html(), {hintText: encodeURI(hint), visual_hintText: hint}));
```

Comment:
Two issues: 

1. hintText is not hintText, it is some form of ID. The name should indicate that function (e.g. hintIdentifier or similar -- you know better how it is used than I do). If anything, visual_ isn't necessary. 
2. Use proper conventions on camelcase vs. underscores. You're mixing conventions in one variable